src/content/docs/guides/usage/control-busses.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Route effects with control busses"
+title: "Modify sounds while they are playing with control busses"
```

Comment:
Maybe just "Modify sounds with control busses" to be a bit more succinct?

Reply:
It's true that the title is maybe too long, but "modifying sounds" is what most control patterns do... How about `Modify ongoing sounds with control busses`?

Reply:
Ok yeah that makes sense :sweat_smile: